variables.tf

```diff
@@ -94,27 +93,52 @@ variable "topics" {
   default     = []
 }
 
-variable "gitignore_template" {
-  description = "Gitignore template to include in the repository"
+variable "visibility" {
+  description = "Visibility of the repository. One of: `public`, `private`, or `internal`"
   type        = string
-  default     = null
-}
-
-variable "access_teams" {
-  description = "List of maps with the `name` of the team that can access the repository and its `permission` (e.g., [{name = test, permission = pull}])"
-  type        = list(any)
-  default     = []
-}
-
-variable "restricted_pr_teams" {
-  description = "List of organizational teams who can dismiss a pull request"
-  type        = list(string)
-  default     = []
+  default     = "public"
 }
 
-variable "restricted_push_teams" {
-  description = "List of organizational teams who can push to the branch"
-  type        = list(string)
-  default     = []
+variable "vulnerability_alerts" {
+  description = "Whether to enable security alerts for dependencies"
+  type        = bool
+  default     = true
+}
+
+variable "template" {
```

Comment:
It may make sense to change variable "template" to "templates" since it is a list of objects.

Reply:
Good eyes. The type is actually wrong. Only a single template is supported. I'll change the type to object({...}) instead of list(object({...})).

Reply:
@mcarmean-novetta fixed.